src/client/datascience/notebook/outputs/plotViewHandler.ts

```diff
@@ -49,7 +49,7 @@ function getOutputItem(editor: NotebookEditor, outputId: string, mimeType: strin
 
 // Wrap our PNG data into an SVG element so what we can display it in the current plot viewer
 function convertPngToSvg(pngOutput: NotebookCellOutputItem): string {
-    const imageBuffer = Buffer.from(pngOutput.data.buffer); // .buffer to not make a copy
+    const imageBuffer = Buffer.from(pngOutput.data);
```

Comment:
Maybe if someone on the team has more buffer expertise they could pick out what I did wrong here. From the docs I expected the data.buffer method to work, and I believe that I was testing locally with it working fine. But after pulling the most recent VS Code + main bits I was getting unexpected data from this versus what I was seeing before, while just using the data was working correctly.